### ISSUE TITLE
Standardize icon structure for consistent color overrides

### ISSUE THREAD

**Issue by GBKS** [GBKS/bitcoin-wallet-ui-kit#14]
When switching an icon instance out, the new icon you pick should maintain the color overrides you added to the previous one. This does not currently happen because the icon layers have different structures, so Figma does not know how to maintain the colors.

Chris researched and the solution appears to be to turn the content of every icon frame into a union an give it the same name.

It may be possible to automate this via a script.

This should be done after #13.

**Comment by chrislueders** [comment]
This plugin can help us to speed up the process:
https://www.figma.com/community/plugin/1196488492656121055/Rename-childs

How to:
https://www.youtube.com/watch?v=zddMRhG7tYs

**Comment by editwentyone** [comment]
is it possible, that this task is already done? I duplicated the file and ran the plugin to rename everything to "Shape" and it worked flawlessly. 

another note: I would keep the matrix as one whole source of truth and keep track inside a note on the side, when something was imported.

also separate WIP icons that are not ready yet more clearly and maybe put a dot in front of the name. just some thoughts, maybe it helps a bit
